src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreationParameters.java

```diff
@@ -17,14 +20,21 @@
 
     private final boolean forceA1Var;
 
+    private final BranchesCurrentManager branchesCurrentManager;
```

Comment:
`BranchesCurrentManager` is probably too much for just a branch list filtering, why not just a list of branch id and empty (or null) means all branches?